guides/plugins/plugins/framework/store-api/add-store-api-route.md

```diff
@@ -61,7 +61,7 @@ use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
 use Shopware\Core\Framework\Plugin\Exception\DecorationPatternException;
 use Shopware\Core\Framework\Routing\Annotation\Entity;
 use Shopware\Core\System\SalesChannel\SalesChannelContext;
-use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Attribute\Route;
```

Comment:
Great that this article is up to date then 🙂 .

But I guess there is still work for many other articles, right? Just search for this use statement in this repository:
https://github.com/search?q=repo%3Ashopware%2Fdocs%20use%20Symfony%5CComponent%5CRouting%5CAnnotation%5CRoute%3B&type=code

I find 17 files that still use this, now outdated, way of annotation 🙈

Reply:
I will have a look, if I could find and fix other files :+1:

Reply:
@MalteJanz @Isengo1989 I found some other places :blush: Pls have a look :blue_heart: